### PR TITLE
Home work 3

### PR DESCRIPTION
**Pull request by ohlabiak**

### CODE REVIEW

**Comment by ohlabiak** [comment]
sorry about commit "add" I forgote it first, but last times i was trying write right commit

File: src/App.js [hunk 1]
```diff
@@ -1,18 +1,160 @@
-import './App.css';
+import css from './App.css';
```
Comment (AgentIvan) [inline src/App.js:1]: The wrong import of module

File: src/components/PokemonCard/PokemonCard.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
+import {useState} from "react";
+
+import css from './style.module.css';
+import cardBackSide from './assets/card-back-side.jpg';
+
+const PokemonCard = ({name, img, id, type, values}) => {
+    const [isActive, setActive] = useState(false);
```
Comment (AgentIvan) [inline src/components/PokemonCard/PokemonCard.js:7]: This state is never used

**Review by AgentIvan — COMMENTED** [review]
Isn't approved
Please correct commented before

**Pull request merged**